src/styles/blocks/card.scss

```
@@ -0,0 +1,105 @@
+/* code here */
+@import "../utils/mixins";
+
+.product {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  width: $card-width;
+
+  &:nth-child(-n + 4) {
+    @include margin-nth-child;
+
+    margin-right: $card-margin-right;
+  }
+
+  &:nth-last-child(-n + 4) {
+    @include margin-last-child;
+
+    margin-right: $card-margin-right;
+  }
+
+  &:nth-child(4n) {
+    @include margin-r-last-child;
+  }
+
+  &__photo {
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+  &__description {
```

```suggestion
  }
  
  &__description {
```

Add spaces between rules